source/blog/2016-07-20-vagrant-service-manager-install-cli.html.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Download client binaries through vagrant-service-manager in ADB
+author: budhram
+date: 2016-07-20 09:56 UTC
+tags: atomic-developer-bundle, vagrant-service-manager
+published: true
+comments: true
+---
+
+The [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) plugin is designed to enable easier access to the features and services provided by the [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). More information can be found in the README  of 'vagrant-service-manager' repo.
+
+This blog is about the new feature in 'vagrant-service-manger' i.e. `install-cli` command. `install-cli` got released with 'vagrant-service-manager' version 1.2.0. This command installs the client binary for any of the services provided by ADB. Currently, 'docker' and 'openshift' are supported.
+
+The `install-cli` command syntax looks like as below:
+
+```
+vagrant service-manager install-cli [service]
+
+Where ‘service’ can be ‘docker’ and ‘openshift’.
+```
+
+Make sure you have 'vagrant-service-manager' plugin installed in your machine. If not, install it via following command as:
+
+```
+$ vagrant plugin install vagrant-service-manager
+```
+
+Now, ensure it has been installed properly with version 1.2.0 by running following command as:
+
+```
+$ vagrant plugin list
+vagrant-service-manager (1.2.0)
+```
+
+Before proceeding let’s setup the ADB development environment. I am using [OpenShift Vagrantfile](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/centos/centos-openshift-setup/Vagrantfile) for my development environment. If you are too lazy to navigate to ADB repo and get the respective Vagrantfile, then run the following command:
+
+```
+$ wget https://raw.githubusercontent.com/projectatomic/adb-atomic-developer-bundle/master/components/centos/centos-openshift-setup/Vagrantfile
+```
+
+Now, do `vagrant up`. Be patience here as it will take some time in downloading [ADB box](https://atlas.hashicorp.com/projectatomic/boxes/adb) along with docker images for OpenShift. Once this process is over, you will see the nice message of accessing OpenShift web console and server login through 'oc' client.
```

Comment:
`vagrant up` will take few minutes to 30 minutes as it downloads OpenShift Origin docker images from docker hub. Post the command completion you should see information to access OpenShift setup by ADB.